var/spack/repos/builtin/packages/py-uwsgi/package.py

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Missing a license

Reply:
What sort of license is a recipe supposed to have?